pom.xml

```diff
@@ -89,6 +89,7 @@
         <version.org.wildfly.checkstyle-config>1.0.8.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.client.config>1.0.1.Final</version.org.wildfly.client.config>
         <version.org.wildfly.common>1.6.0.Final</version.org.wildfly.common>
+        <version.org.wildfly.core>23.0.0.Beta4</version.org.wildfly.core>
```

Comment:
WildFly Core depends on Elytron. We can't add this dependency here (see my comment above).

Reply:
removed circular dependency. Introduced new exception classes instead and got rid of server specific code/classes.